### PR TITLE
fix(i18n/ar): update Reading Time localization for Arabic language 

### PR DESCRIPTION
**Pull request by Raddah**
readingTime variable updated to be more clear for Arabic readers

The quantity of `1` in  Arabic doesn't need  to be counted so, I have removed the count variable for `One`, there is a word to describe that specifically. 

image

Other variable localization updated to match Arabic grammar tenses.

image

### CODE REVIEW

**Review by fariszr — APPROVED** [review]

File: i18n/ar.yaml [hunk 1]
```diff
@@ -30,8 +30,8 @@ article:
         other: التعديل الاخير
 
     readingTime:
-        one: "قرائة {{ .Count }} دقيقة"
-        other: "قرائة {{ .Count }} دقائق"
+        one: " تُقرأ خلال دقيقة"
```
Comment (CaiJimmy) [inline i18n/ar.yaml:33]: I don't know Arabic. Is that leading space necessary?

**Comment by fariszr** [comment]
Nope, Its a typo.

-------- Original Message --------

**Comment by fariszr** [comment]
Anything stopping this from being merged ?
I think you have the permission to edit the PR to remove the space.

**Review by CaiJimmy — APPROVED** [review]

**Comment by CaiJimmy** [comment]
Merged, thanks!

**Pull request merged**